docs/ss_slide_006.md

```diff
@@ -1,12 +1,12 @@
 # Patterns can also exhibit qualitative behavioral differences at different step sizes
 
 {:style="text-align:center;"}
-![frog behavior changes at different step sizes](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/consequential_step_size/supplemental_item_1_step_size_behavior.gif)
+![frog behavior changes at different step sizes](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/consequential_step_size/supplemental_item_1_step_size_behavior.gif)
 
 ### This 'frog' pattern in s613, a glaberish CA exhibits qualitatively variable behavior across a range of step sizes. 
 {:style="text-align:center;"}
 
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/ss_slide_005.md) -- [Next slide](https://rivesunder.github.io/yuca/ss_slide_007)
+[Previous slide](https://rivesunder.github.io/yuca_docs/ss_slide_005.md) -- [Next slide](https://rivesunder.github.io/yuca_docs/ss_slide_007)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.